.openpublishing.redirection.json

```diff
@@ -790,7 +790,22 @@ Old redirects (to /archive/)
       /* Extensions: */
       {
         "source_path": "microsoft-edge/extensions/microsoft-browser-extension-policy.md",
-        "redirect_url": "https://learn.microsoft.com/legal/microsoft-edge/microsoft-browser-extension-policy",
+        "redirect_url": "https://learn.microsoft.com/legal/microsoft-edge/extensions/microsoft-browser-extension-policy",
+        "redirect_document_id": false
+      },
+      {
+        "source_path": "microsoft-edge/extensions-chromium/store-policies/csp.md",
+        "redirect_url": "https://learn.microsoft.com/legal/microsoft-edge/extensions/csp",
+        "redirect_document_id": false
+      },
```

Comment:
This redirect rule should be removed, since we're keeping the CSP article here in this docset.

Reply:
removed csp redirect